srcpkgs/vit/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'vit'
+pkgname=vit
+version=2.1.0
+revision=0
+build_style=python3-module
+hostmakedepends="python3-pip task python3-pytz python3-urwid python3-tzlocal"
+depends="task"
+checkdepends=
+short_desc="VIT is a lightweight, fast, curses-based front end to Taskwarrior "
+maintainer="pinoinha <pinoinha@disroot.org"
+license="MIT"
+homepage="https://github.com/vit-project/vit"
+changelog="https://raw.githubusercontent.com/vit-project/vit/2.x/CHANGES.md"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
+checksum=56676651164fb71878a0c1e4bb369cfa336d46e6fd0de7e6167b57fca7eaf662
+
+pre_build() {
+	pip3 install tasklib
+}
```

Comment:
Why is this necessary? It's not acceptable to run pip in package templates, if we need to package tasklib, it should be done in a separate commit in this PR.

Reply:
that's a good point. I'll package it and reopen this PR afterwards

Reply:
You don't need to close this PR